Cargo.toml

```
@@ -11,6 +11,8 @@ anyhow = "1.0"
 chrono = "0.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+serde_yaml = "0.8"
+lazy_static = "1.4.0"
```

You don't need to pin the version of this crate:

```suggestion
lazy_static = "1.4"
```